pkg/sync/v6/data/ConfigMap/openshift-azure-monitoring/metrics-bridge.yaml

```diff
@@ -5,6 +5,8 @@ data:
     statsdSocket: /var/etw/mdm_statsd.socket
     queries:
     - query: 'ALERTS'
+    - query: 'ALERTS{alertname="DeadMansSwitch",alertstate="firing"}'
+      name: osacluster_monitored
```

Comment:
already discussed with @Makdaam regarding the name `osacluster_monitored`. Not sure it sounds right. What do the others think?